packages/data-seeder/src/locations.ts

```diff
@@ -227,31 +227,48 @@ export async function seedLocations(token: string) {
   const locations = (await getLocations()).filter((location) => {
     return !savedLocationsSet.has(location.id)
   })
-  const res = await fetch(`${GATEWAY_HOST}/location?`, {
-    method: 'POST',
-    headers: {
-      Authorization: `Bearer ${token}`,
-      'Content-Type': 'application/fhir+json'
-    },
-    body: JSON.stringify(
-      locations
-        // statisticalID & code are legacy properties
-        .map(({ id, locationType, ...loc }) => ({
-          statisticalID: id,
-          code: locationType,
-          ...loc
-        }))
+  const locationSegments: TypeOf<typeof LocationSchema>[] = []
+  jurisdictionLevels.forEach((jurisictionType) => {
+    const jurisictionTypeLocations = locations.filter(
+      (location) =>
+        location.jurisdictionType === jurisictionType &&
+        location.locationType === 'ADMIN_STRUCTURE'
     )
+    if (jurisictionTypeLocations.length) {
+      locationSegments.push(jurisictionTypeLocations)
+    }
```

Comment:
Does this make sure that the locations are in the order we want, i.e. STATE -> DISTRICTS ... ?

Reply:
I guess it does as we are traversing the jurisdictionLevels array in that order